lib/rules/generator-star.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileoverview Rule to check for the position of the * in your generator functions
+ * @author Jamund Ferguson
+ * @copyright 2014 Jamund Ferguson. All rights reserved.
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+    var position = context.options[0];
+
+    // TODO: find out the right way to handle this
+    // basically fail silently if they don'"t include a proper position
+    if (["start", "end"].indexOf(position) === -1) {
```

Comment:
Will default to `end` in the case that nothing is passed, as that's the style you seem to use the most in understanding es6.